src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
```

Comment:
Don't use abstract classes to set behavior for classes. In this case, since `Figure` does not provide any default implementations and is used to define a type hierarchy, an interface would be more appropriate.